container-registry/aws/ecr/examples/complete/versions.tf

```diff
@@ -9,7 +9,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 5.4.0"
+      version = ">= 5.4.0"
```

Comment:
why do you relaxe this version? Are you sure that a major version will be compatible?

Reply:
from @lemaitre-aneo  previous comment, it is best practice to not put it at the module level but at the root module level